tale_studio/tgi_wrapper.py

```
@@ -0,0 +1,27 @@
+    messages: List[Dict[str, str]],
+    model_settings: ModelSettings,
+    url: str = DEFAULT_URL,
+):
+    prompt = format_template(messages, model_settings.prompt_template)
+    params = vars(model_settings.generation_params)
+    data = {
+        "inputs": prompt,
+        "parameters": {"do_sample": True, "seed": 42, "watermark": False, **params},
+    }
+    headers = {"Content-Type": "application/json"}
+    response = requests.post(url=url, json=data, headers=headers)
```

Ruff (0.14.4)

Probable use of `requests` call without timeout

(S113)